docs/prow/authorization.md

```diff
@@ -2,10 +2,45 @@
 
 ## Required GCP Permissions
 
-Every Prow clsuter should be deployed in its own GCP project. You need an owner access to deploy Prow and configure it properly.
+Every Prow cluster should be deployed in its own GCP project. You need an owner access to deploy Prow and configure it properly. In addition you need following service accounts created:
+
+- sa-gcs-plank - service account used by Prow plan microservice with the `Storage Object Admin` role
```

Comment:
| Service account name | description | required roles |
|:--------------------:|:-----------:|:--------------:|
| sa-gcs-plank | used by Prow plan microservice | Storage Object Admin |

Reply:
Please enclose the SA data in a chart like this

Reply:
done